apps/apps.c

```diff
@@ -2294,29 +2294,28 @@ int app_access(const char* name, int flag)
 #ifdef _WIN32
 int app_isdir(const char *name)
 {
-    HANDLE hList;
-    WIN32_FIND_DATA FileData;
+    DWORD attr;
 # if defined(UNICODE) || defined(_UNICODE)
     size_t i, len_0 = strlen(name) + 1;
+    WCHAR tempname[MAX_PATH];
 
-    if (len_0 > OSSL_NELEM(FileData.cFileName))
+    if (len_0 > MAX_PATH)
         return -1;
 
 #  if !defined(_WIN32_WCE) || _WIN32_WCE>=101
     if (!MultiByteToWideChar
```

Comment:
If you modify code that was malformed in grand reformat, it's appropriate to fix it at once. In this case
```
if (!MultiByteToWideChar(CP_ACP, 0, ...)
```